hedyorg/hedy
# [UI] Add a New Year's story to Level 11 Quiz

**Description**

Adds a New Year's story to Level 11 Quiz

**Fixes issue #3455**

**How to test**

- Go to http://localhost:8080/hedy/11#quiz
- Click "go_to_first_question" button
- See that the New Year's question appears and that clicking and submitting an answer allows for either feedback or the appearance of the message "correct_answer"

**Checklist**  
- [X ] Describes changes in the format above
- [X ] Links to an existing issue or discussion 
- [ X] Has a "How to test" section
